src/sentry/incidents/endpoints/organization_alert_rule_available_action_index.py

```diff
@@ -65,19 +57,21 @@ def get(self, request, organization):
 
         actions = []
 
-        integrations = get_available_action_integrations_for_org(organization).order_by("id")
+        # Cache Integration objects in this data structure to save DB calls.
         provider_integrations = defaultdict(list)
-        for integration in integrations:
+        for integration in get_available_action_integrations_for_org(organization):
             provider_integrations[integration.provider].append(integration)
-        registered_types = AlertRuleTriggerAction.get_registered_types()
-        registered_types.sort(key=lambda x: x.slug)
```

Comment:
was this important to keep? I'm not sure what the purpose of sorting by slug is for the registered_type

Reply:
`registered_types` isn't being read after the `sort` call so I figured we could skip it.

Reply:
Oops, I guess I was intending to use this to iterate over and forgot to do so. I guess the frontend applies it's own sorting to the list anyway, so probably no need to do it here.